index.js

```diff
@@ -34,6 +39,22 @@ var cli = new Liftoff({
   completions: completion,
   extensions: interpret.jsVariants,
   v8flags: v8flags,
+  configFiles: {
+    '.gulprc': {
+      home: {
+        path: '~',
+        extensions: interpret.extensions,
```

Comment:
Good catch!  I actually want to write my configs as `.gulp.yml` which also isn't supported in `jsVariants`.  Please leave it as `interpret.extensions`.